build_push.sh

```diff
@@ -1,21 +1,17 @@
 #!/usr/bin/env sh
 
-builds=$(echo '
-9.6.10    9.6.10-r0   3.6
-9.6       9.6.10-r0   3.6
-9         9.6.10-r0   3.6
-latest    9.6.10-r0   3.6
-' | grep -v '^#' | tr -s ' ')
+builds=\
+"10.10_10.10-r0_3.8",\
+"10_10.10-r0_3.8",\
+"latest_10.10-r0_3.8"
```

Comment:
So this is a lot more confusing to read than the old version. Why switch format?

Reply:
By doing it this way we don't have to touch the $IFS variable.
On my system (Arch Linux), setting `IFS=$'\n'` mangles the value of `$build` when I run this script.